referrer-policy/generic/subresource/xhr.py

```diff
@@ -0,0 +1,13 @@
+import json
```

Comment:
This file is already being used for both:
```json
 "subresource_path": {   
   ...
   "fetch-request": "/referrer-policy/generic/subresource/xhr.py",
   ...
   "xhr-request": "/referrer-policy/generic/subresource/xhr.py"
  }
```

Let's consider the subresources directory and naming:

```
subresource
├── document.py
├── image.py
├── script.py
├── worker.py
└── xhr.py
```

Against:

```
subresource
├── document.py
├── image.py
├── script.py
├── worker.py
└── echo-headers.py
```

Fetch is still very associative with XHR so that's my reasoning. :-)
Other scripts also echo the headers, just in a slightly different fashion.